google-datacatalog-saphana-connector/src/google/datacatalog_connectors/saphana/datacatalog_cli.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/python
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import sys
+
+from google.datacatalog_connectors.rdbms import\
+    datacatalog_cli
+from google.datacatalog_connectors.saphana.scrape import metadata_scraper
+
+
+class SapHana2DatacatalogCli(datacatalog_cli.DatacatalogCli):
+
+    def _get_metadata_scraper(self):
+        return metadata_scraper.MetadataScraper
+
+    def _get_host_arg(self, args):
+        return args.saphana_host
+
+    def _get_connection_args(self, args):
+        return {
+            'database': args.saphana_database,
+            'host': args.saphana_host,
+            'user': args.saphana_user,
+            'pass': args.saphana_pass
+        }
+
+    def _get_entry_group_id(self, args):
+        return args.datacatalog_entry_group_id or 'saphana'
+
+    def _get_metadata_definition_path(self):
+        return os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                            'config/metadata_definition.json')
+
+    def _get_query_path(self, args):
+        return os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                            'config/metadata_query.sql')
+
+    def _parse_args(self, argv):
+        parser = argparse.ArgumentParser(
+            description='Command line to sync saphana '
```

Comment:
```suggestion
            description='Command line to sync SAP Hana '
```
The official product name is `SAP Hana`, so I recommend using it in any kind of documentation and messages. WDYT? 🤔